pipelines/hyperkube-release/Jenkinsfile

```diff
@@ -0,0 +1,31 @@
+def bash(String cmd) { sh("#/usr/bin/env bash\nset -euo pipefail\n${cmd}") }
+
+podTemplate(label: 'hyperkube-build',
+  containers: [containerTemplate(name: 'build', image: 'quay.io/coreos/hyperkube-builder:0.2', ttyEnabled: true, command: 'cat')],
+  volumes: [hostPathVolume(hostPath: '/var/run/docker.sock', mountPath: '/var/run/docker.sock')]
+) {
+  node('hyperkube-build') {
+    stage('stage 1') {
+      git 'https://github.com/coreos/kubernetes.git'
+      container('build') {
+        withCredentials([usernamePassword(credentialsId: 'quay_userpass', usernameVariable: 'DOCKER_USER', passwordVariable: 'DOCKER_PASS')]) {
+          bash '''
+              export IMAGE_TAG=$(echo ${KUBERNETES_VERSION} | tr + _)
```

Comment:
What does `tr + _` do? I assume it munges up the sha1 hash into something that would work for a image tag. Does this work for all of the tags that we create? (and the result of running it through `tr + _` is the same as what we label manually?

(If this is the same logic that was being done before in the scripts, just let me know and ignore my comments I guess, just wanted to check...)

Reply:
Oh hey, it looks like this is instead of the long 2- script where we had lots of unknowns. Cool.

Reply:
correct. deleted the 2- script... the tr + _ changes, 1.10.0+coreos.0 to 1.10.0_coreos.0 which are the image names on quay.